extensions/qute/deployment/src/main/java/io/quarkus/qute/deployment/MessageBundleProcessor.java

```diff
@@ -1041,8 +1035,7 @@ public boolean test(String name) {
             }
             // E.g. to match the bundle class generated for a localized file; org.acme.Foo_en -> org.acme.Foo
             className = additionalClassNameSanitizer.apply(className);
-            return applicationArchives.containingArchive(className) != null
-                    || GeneratedClassGizmoAdaptor.isApplicationClass(name);
+            return GeneratedClassGizmoAdaptor.isApplicationClass(className);
```

Comment:
The main part of the fix is `GeneratedClassGizmoAdaptor.isApplicationClass(name)` -> `GeneratedClassGizmoAdaptor.isApplicationClass(className)`!